fauna/client/client.py

```diff
@@ -523,6 +531,7 @@ def _next_element(self):
           FaunaError.parse_error_and_throw(event, 400)
 
         self.last_ts = event["txn_ts"]
+        self.last_cursor = event["cursor"]
```

Comment:
looking the tests failures, looks like `cursor` is not present in all events, you might need to change to `event.get("cursor")`

Reply:
Yeah. That and also I forgot to update the mocked tests somehow.